tendrl/commons/tests/objects/global_details/test_global_details.py

```diff
@@ -0,0 +1,96 @@
+import __builtin__
+import maps
+from mock import patch
+
+from tendrl.commons.objects.cluster_tendrl_context import ClusterTendrlContext
+from tendrl.commons.objects.global_details import GlobalDetails
+from tendrl.commons.utils import etcd_utils
+
+
+# tests __init__ and render
+def test_render():
+    setattr(__builtin__, "NS", maps.NamedDict())
+    NS.tendrl_context = maps.NamedDict()
+    NS.tendrl_context.integration_id = "test_integration_id"
+
+    # IMPORTANT: Without the following two lines, this tests somehow
+    # causes an error in "test_gluster_peer_init.py"
+    # Find out why
```

Comment:
Add todos in below format 

"# TODO ($username) Without the following two lines, this tests somehow
#causes an error in "test_gluster_peer_init.py"
#Find out why
"